tests/manage/monitoring/test_workload_fixture.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf8 -*-
+import logging
+
+import pytest
+
+from ocs_ci.utility.prometheus import PrometheusAPI
+# from ocs_ci.ocs.ocp import OCP
+# from tests import helpers
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.libtest
+def test_workload_rbd(workload_storageutilization_50p_rbd):
+    logger.info(workload_storageutilization_50p_rbd)
+    prometheus = PrometheusAPI()
+
+
+@pytest.mark.libtest
+def test_workload_rbd_again(workload_storageutilization_50p_rbd):
```

Comment:
This probably won't start second utilization test with the same workload fixture as there should be already gathered results after the execution of the first one. How do you ensure order so that `test_workload_rbd_again` is executed after `test_workload_rbd` as name suggests?

Reply:
You are right that the workload will be executed only once, but that's the point. The idea behind `test_workload_fixture.py` is to demonstrate basic functionality of the workload fixtures, including the fact that when the same fixture is used again, only single workload is performed.

I will rename this function and add some comment about this though.